modules/standard/Random.chpl

```diff
@@ -667,11 +843,21 @@ module Random {
         //if arr.domain.type.strideable then
         //  compilerError("Shuffle requires non-strideable 1-D array");
 
+        if parSafe then
```

Comment:
Or, it would've been another good thing to talk about as a design decision rather than part of a code review.